src/common/webview.cpp

```diff
@@ -224,6 +225,51 @@ wxString wxWebView::GetUserAgent() const
     return userAgent;
 }
 
+bool wxWebView::RunScript(const wxString& javascript, wxString* output) const
+{
+    m_syncScriptResult = -1;
+    m_syncScriptOutput.Clear();
+    RunScriptAsync(javascript, (void*)this);
```

Comment:
The cast is not really needed, is it?
```suggestion
    RunScriptAsync(javascript, this);
```

Reply:
I'll check but I think it's required because of `const` but it isn't used anymore anyway, so I'll just remove it.